google/cloud/forseti/services/inventory/storage.py

```diff
@@ -758,6 +758,7 @@ def clear_cai_data(session):
         except SQLAlchemyError as e:
             LOGGER.exception('Attempt to delete data from CAI temporary store '
                              'failed, disabling the use of CAI: %s', e)
+            session.rollback()
```

Comment:
Thanks for this... is this the only place where the rollback is needed?  Are there others?